### DISCUSSION TITLE
VB6/VBA to VS Code pain points

### DISCUSSION THREAD

**Discussion by WaynePhillipsEA** [twinbasic/twinbasic#297] [General]
Let me start by saying I absolutely love VS Code.

However, I've heard several comments over the last few months that some people that aren't familiar with VS Code are struggling with the change from VB6/VBA to VS Code/twinBASIC.  So if you fit in that category, please give us your insights as to what you found difficult here.   If you have any ideas on what we might be able to do to help with the transition, please do let us know.

**Comment by eantar** [comment]
Hey Wayne thanks for starting this discussion thread. When I get a chance I will post exactly the issue I was having. But as a summary, I tried to open one of the workspaces for a DLL I created. But all I could see was what looked like C code, I couldn't find the VBA Code that I had written. in VS Code. This is just a general description. I'll try and post something more useful in the future.

I think it would be great if, once completed, you guys did like a tour! I would join that for sure.

**Reply by WaynePhillipsEA** [reply]
No problem.  If you encounter that again, it would be really useful to see a screenshot if possible.

A video tour where we compare the VBIDE features side by side with the twinBASIC equivalents might be useful.
